pkg/gatherers/clusterconfig/pod_disruption_budgets.go

```diff
@@ -11,21 +12,33 @@ import (
 )
 
 const (
-	gatherPodDisruptionBudgetLimit = 5000
+	gatherPodDisruptionBudgetLimit = 100
 )
 
 // GatherPodDisruptionBudgets gathers the cluster's PodDisruptionBudgets.
 //
 // The Kubernetes api https://github.com/kubernetes/client-go/blob/v11.0.0/kubernetes/typed/policy/v1beta1/poddisruptionbudget.go#L80
 // Response see https://docs.okd.io/latest/rest_api/policy_apis/poddisruptionbudget-policy-v1beta1.html
 //
-// * Location in archive: config/pdbs/
-// * See: docs/insights-archive-sample/config/pdbs
-// * Id in config: clusterconfig/pdbs
-// * Since versions:
-//   - 4.4.30+
-//   - 4.5.34+
-//   - 4.6+
+// ### Sample data
+// - docs/insights-archive-sample/config/pdbs/openshift-machine-config-operator/etcd-quorum-guard.json
+//
+// ### Location in archive
+// - `config/pdbs/{namespace}/{name}.json`
+//
+// ### Config ID
+// `clusterconfig/pdbs`
+//
+// ### Released version
+// - 4.6.0
+//
+// ### Backported versions
+// - 4.4.30+
+// - 4.5.15+
+//
+// ### Changes
+// - The gatherer was changed to gather pdbs only from namespaces with "openshift" prefix
+// and the limit of gathered records to 100 since 4.14.
```

Comment:
I know it's a little anoying, but I think it would be good to keep the original formatting here.